src/main/java/com/epam/test_generator/dto/CaseDTO.java

```diff
@@ -159,14 +174,15 @@ public int hashCode() {
     @Override
     public String toString() {
         return "CaseDTO{" +
-            "id=" + id +
-            ", description='" + description + '\'' +
-            ", steps=" + steps +
-            ", creationDate=" + creationDate +
-            ", priority=" + priority +
-            ", tags='" + tags + '\'' +
-            ", status=" + status +
-            '}';
+                "id=" + id +
```

Comment:
Instead of concatenation I'd recommend using  String.format

Reply:
Done